dd-sdk-android/src/test/kotlin/com/datadog/android/DatadogTest.kt

```diff
@@ -227,6 +231,51 @@ internal class DatadogTest {
         verify(CoreFeature.trackingConsentProvider).setConsent(fakeConsent)
     }
 
+    @Test
+    fun `M no changes W initialize() { verboseDebug } in non debug mode`() {
+        val credentials = Credentials(fakeToken, fakeEnvName, fakeVariant, fakeApplicationId, null)
```

Comment:
Out of consistency, can you also make sure the `ApplicationInfo.FLAG_DEBUGGABLE` is not present in `appContext.fakeAppInfo.flags`. Even though it's initialized by default to `0`, it'd be safer to ensure it's in the state you expect.